action/src/config.ts

```diff
@@ -11,14 +17,16 @@ export const config : Config = {
   },
 };
 
+export const config : Config = getYamlInput('config') as Config || defaultConfig; // Default to default config if not provided
```

Comment:
U can not cat url `string` to `URL`

Reply:
I noticed. I managed to make it work but I am not sure about if its the most effective way to do it. Let me know what you think.